### PR TITLE
Add documentation about Scaleway credentials

### PR DESCRIPTION
**Pull request by remyleone**
Documentation Pull Request

## Description

This PR adds a link towards the credentials page where the tokens are generated in the Scaleway console.

## How Has This Been Tested?

Documentation Pull Request

### CODE REVIEW

**Comment by remyleone** [comment]
Should we add something in the documentation related to #1410 ?

**Comment by jackivanov** [comment]
No, no need to add anything about the modules. Although, we would need to change the link [here](https://github.com/trailofbits/algo/blob/master/roles/cloud-scaleway/tasks/prompts.yml#L4) to https://trailofbits.github.io/algo/cloud-scaleway.html which is coming from this PR

**Comment by remyleone** [comment]
@jackivanov done.

**Comment by remyleone** [comment]
@jackivanov done.

**Comment by jackivanov** [comment]
Thanks @remyleone! Could you please fix the conflicts and we'll merge it?

**Comment by remyleone** [comment]
@jackivanov Done. Do you have further remarks?

File: roles/cloud-scaleway/tasks/prompts.yml [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-      Enter your auth token (https://www.scaleway.com/docs/generate-an-api-token/)
+      Enter your auth token (https://trailofbits.github.io/algo/cloud-scaleway.html)
     echo: false
   register: _scaleway_token
-  when:
-    - scaleway_token is undefined
-    - lookup('env','SCW_TOKEN')|length <= 0
+  when: scaleway_token is undefined
+
+- pause:
```
Comment (jackivanov) [inline roles/cloud-scaleway/tasks/prompts.yml]: It was removed in #1410

File: roles/cloud-scaleway/tasks/prompts.yml [hunk 2]
```diff
@@ -1,12 +1,16 @@
 ---
 - pause:
     prompt: |
-      Enter your auth token (https://www.scaleway.com/docs/generate-an-api-token/)
+      Enter your auth token (https://trailofbits.github.io/algo/cloud-scaleway.html)
     echo: false
   register: _scaleway_token
-  when:
-    - scaleway_token is undefined
-    - lookup('env','SCW_TOKEN')|length <= 0
```
Comment (jackivanov) [inline roles/cloud-scaleway/tasks/prompts.yml]: We need to keep both checks, I guess this is a mistake after rebasing?

**Comment by remyleone** [comment]
@jackivanov Sorry it was a rebase mistake. It is fixed now.

**Pull request merged**